app/controllers/edi_lins_controller.rb

```diff
@@ -7,10 +7,49 @@ def allow_nobib
     end
   end
 
-  def update
-    @message = EdiLin.make_updates(params[:vendors], params[:invoice_number], params[:invoice_line_number])
-    flash[@message[0].to_sym] = @message[1]
+  def fix_duplicate_barcode
+    respond_to do |format|
+      format.js # modal window: fix_duplicate_barcode.js.erb
+    end
+  end
+
+  def show
+    @edi_lin = EdiLin.where(barcode_num: params[:barcode_num])
+  end
+
+  def index
+    @edi_lin = EdiLin.where(barcode_num: params[:barcode_num])
+    message = "Barcode #{params[:barcode_num]} is not a duplicate. It only appears once in the EDI LINE table."
+    if @edi_lin.size < 2
```

Comment:
Since this is just a test you can easily make (edi_lin.size), can we test for both 0 and <2 ?
Message if edi_lin.size = 0 would be 'Barcode #{params[:barcode_num]} does not exist as a barcode in the EDI_LIN table.'
Elsif edi_lin_size < 2
  message = what you have above, but change EDI LINE to EDI_LIN